model/Hardware/Vocabularies/RequirementsStatusType.md

```diff
@@ -0,0 +1,22 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# RequirementsStatusType
+
+## Summary
+
+Requirements go through different status types during their lifecycle.
+
+## Description
+
+Requirements go through different status types during their lifecycle. Below are common types categorized based on their progress and decision status.
+
+## Metadata
+
+- name: RequirementsStatusType
+
+## Entries
+
+- Draft: Initial requirement suggestion, not yet reviewed.
+- ReadforReview: Being refined and not yet finalized .
+- Approved: Reviewed and formally accepted for development.
+- Retired: Obsolete or superseded requirement
```

Comment:
```suggestion
- draft: Initial requirement suggestion, not yet reviewed.
- readForReview: Being refined and not yet finalized.
- approved: Reviewed and formally accepted for development.
- retired: Obsolete or superseded requirement.
```

Base on other vocabs, the casing convention for an entry name is [lowerCamelCase](https://en.wikipedia.org/wiki/Camel_case)